src/client/components/ChannelHeadNav/ChannelHeadNav.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+export default function(channelObj, urlBack) {
+  return (
+    <div>
+      <Link to={urlBack}>Back</Link>
+      <img src={channelObj.imgUrl} alt="" />
+      <h2>{channelObj.name}</h2>
```

Comment:
Same with the name. 

so then you would use it like:

`<Header-navigation name="asd" urlBack="/overview" image="https://abdk.jpeg" />`